torch/compiler/__init__.py

```diff
@@ -1,7 +1,8 @@
 # mypy: allow-untyped-defs
-import io
+from __future__ import annotations
```

Comment:
Was this necessary? annotations is deprecated so we're trying to avoid adding more...